extensions/smallrye-reactive-messaging-kafka/runtime/src/main/java/io/quarkus/smallrye/reactivemessaging/kafka/KafkaConfigCustomizer.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.smallrye.reactivemessaging.kafka;
+
+import java.util.Map;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+import org.eclipse.microprofile.config.Config;
+
+import io.quarkus.kafka.client.tls.QuarkusKafkaSslEngineFactory;
+import io.smallrye.reactive.messaging.ClientCustomizer;
+
+@ApplicationScoped
+public class KafkaConfigCustomizer implements ClientCustomizer<Map<String, Object>> {
+
+    @Override
+    public Map<String, Object> customize(String channel, Config channelConfig, Map<String, Object> config) {
+        // TODO verify other ssl properties
```

Comment:
What do you mean?

Reply:
If other ssl-related Kafka client config has been provided, by configuring the SslEngineFactory we'd simply not use them. If we detect SSL configs in addition to the tls-configuration-name we could log a warning.